plugins/generic/customBlockManager/CustomBlockEditForm.inc.php

```diff
@@ -48,7 +48,7 @@ function CustomBlockEditForm(&$plugin, $journalId) {
 	function initData() {
 		$journalId = $this->journalId;
 		$plugin =& $this->plugin;
-
+		
```

Comment:
There are a couple of extra whitespace changes in the pull request; better remove those just to keep things clean.